csharp/native/SubTasking/Client/ArmoniK.Samples.SubTasking.Client/ArmoniK.Samples.SubTasking.Client.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
I advise you to remove it !
And use dotnet 8 as dotnet 6 will be out of support in less than 6 months.